common/src/main/java/io/seata/common/util/IOUtil.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.common.util;
+
+/**
+ * @author jsbxyyx
+ */
+public class IOUtil {
+
+    /**
+     * close Closeable
+     * @param closeables the closeables
+     */
+    public static void close(AutoCloseable... closeables) {
```

Comment:
We can merge these two methods into one.
public static void close(AutoCloseable closeable,AutoCloseable... moreCloseables);